b02cf9e61b861d10d806dc23b24bcb3f/update/sys_ws_operation_bbcb42a61bc61d10d806dc23b24bcbd2.xml

```diff
@@ -1,26 +1,94 @@
 <?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ws_operation">
     <sys_ws_operation action="INSERT_OR_UPDATE">
         <active>true</active>
-        <consumes>application/json,application/xml,text/xml</consumes>
-        <consumes_customized>false</consumes_customized>
+        <consumes>application/json,application/xml,text/xml,application/x-www-form-urlencoded</consumes>
+        <consumes_customized>true</consumes_customized>
         <default_operation_uri/>
         <enforce_acl>cf9d01d3e73003009d6247e603f6a990</enforce_acl>
         <http_method>POST</http_method>
         <name>Event</name>
-        <operation_script><![CDATA[(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {
-
-    if (request.body.data.challenge){
-		response.setStatus(200);
-		response.setContentType('text/plain');
-		response.getStreamWriter().writeString(request.body.data.challenge);
-	} else {
-		if (request.body.data.event.bot_id) return;
-		//gs.info('slackoff: ' + (request.body.dataString));
```

Comment:
While I totally agree with removing the comment call, removing the return for bot-messages will open up recursive triggers for payloads from the bot.

Can you please re-introduce this? If it was blocking functionality, let us know and we can brainstorm an updated approach